content/docs/sidebar.json

```diff
@@ -35,36 +35,49 @@
     },
     "children": [
       {
-        "slug": "data-and-model-versioning",
-        "tutorials": {
-          "katacoda": "https://katacoda.com/dvc/courses/get-started/versioning"
-        }
-      },
-      {
-        "slug": "data-and-model-access",
-        "tutorials": {
-          "katacoda": "https://katacoda.com/dvc/courses/get-started/accessing"
-        }
-      },
-      {
-        "slug": "data-pipelines",
-        "tutorials": {
-          "katacoda": "https://katacoda.com/dvc/courses/get-started/stages"
-        }
-      },
-      {
-        "label": "Metrics, Parameters, and Plots",
-        "slug": "metrics-parameters-plots",
-        "tutorials": {
-          "katacoda": "https://katacoda.com/dvc/courses/get-started/params-metrics-plots"
-        }
+        "slug": "data",
+        "label": "Data Management",
+        "source": "data/index.md",
+        "children": [
+          {
+            "label": "Accessing Data",
+            "slug": "access",
+            "tutorials": {
+              "katacoda": "https://katacoda.com/dvc/courses/get-started/accessing"
```

Comment:
Let's keep the original page titles for now? I'm not sure there's a goal to rename them and it can become a pretty long discussion. Or is there some need to rename the pages?

Reply:
Let's at least have consistent title, slug, and file names?

Reply:
Also any changes done here should be reflected in the actual MD page titles (H1s) I think.

Reply:
I shortened because sidebar space is limited. Will be reverting not to going into that long discussion 😄